project/.travis.yml.twig

```diff
@@ -75,6 +75,7 @@ matrix:
 
 before_install:
   - git remote add upstream ${UPSTREAM_URL} && git fetch --all
+  - if [[ $lint = 1 ]]; then wget http://cs.sensiolabs.org/download/php-cs-fixer-v2.phar --output-document php-cs-fixer.phar; fi
```

Comment:
Maybe am I missing something, but it looks like the `$lint` var is always undefined.